### DISCUSSION TITLE
No sorting of test results?

### DISCUSSION THREAD

**Discussion by floekke** [testspace-com/support#9] [Q&A]
Hi!

So far my experience with testspace has been smooth.
I've uploaded some junit test results to my project / space. It looks like I cannot sort the test Results on e.g. Duration?
Which effectively makes the Results report unusable for me..

**Comment by munderseth** [comment]
Hi @floekke,

We do not support sorting on duration at this time. 

A few items that might be of interest to you:
- [Failure filtering](https://help.testspace.com/docs/dashboard/space-results#test-failures) - Select the "**n Failures**" button.  We track different types of failures (Flaky, Long-term, etc.)
- You can also `organize` your tests based on your source tree organization using `Folders`. This technique allows better visualization of your test schema. See [here](https://help.testspace.com/docs/publish/push-data-results#source) for details. 

Note that we are currently reviewing improvements to our analytics and *filtering*. If you have any suggestions we would love to hear them.
